### ISSUE TITLE
Difference from word2vec

### ISSUE THREAD

**Issue by renderit** [mattiasostmar/doc2vec_on_usertweets#1]
Hey interesting work here! Could you maybe tak about difference bw this and using tf-idf to compare the tweets?

**Comment by mattiasostmar** [comment]
You could use a mountain of different ways to distinguish people by language e.g. TF-IDF, LDA, LDA2vec etc. There are too many differences in the math, the implemetations etc for me to be able to say anything intelligent about the difference - I experment with different approaches to explore and see what gives interesting results. 

Though, I find TF-IDF to be a human-friendly way of displaying the words that people without prior knowledge about linguistic analysis will think of as describing people.

**Comment by renderit** [comment]
Sure! Thanks.. Yes I was looking for more of an intuitive explaination for the difference bw the 2! :) 
A similar project I am working on involves, 'vectorizing' every user features. Like a `user age, user dob, user blood website, user info and other text` info trying to match/similar with another different vectorized user. I was thinking LDA(topic modeling) to do this, but a lil confused if I should be use doc2vec to convert user info('doc' to vectors) and then process. Any recommendations?

**Comment by mattiasostmar** [comment]
Nah, I should recommend you just experiment with one of them and publish the results online so that others can try it out themselves and learn. :-)
